test/java/org/chromium/net/testing/Http2TestServerTest.java

```diff
@@ -78,6 +103,96 @@ public void getSchemeIsHttps() throws Exception {
     assertThat(response.getEnvoyError()).isNull();
   }
 
+  @Test
+  public void testGetRequest() throws Exception {
+    System.out.println("TEST_testGetRequest");
```

Comment:
nit: presumably we should remove these printlns?

Reply:
done. Though I really feel like it will always be needed. Hopefully there will be a better way to distinguish tests.